Dice.Core/DiceEventArgs.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Dice.Core
+{
+    public class DiceEventArgs : EventArgs
+    {
+        public int FirstDiceSide { get; private set; }
```

Comment:
It's 'die' for the singular. And it's enough to call 'FirstDie', I think - rather obvious.